docs/serving/autoscaling.md

```diff
@@ -179,11 +180,39 @@ template.
 spec:
   template:
     metadata:
-      autoscaling.knative.dev/metric: concurrency
+      autoscaling.knative.dev/metric: cpu
       autoscaling.knative.dev/class: hpa.autoscaling.knative.dev
 ```
 
-## Additional resources
+# Implementing your own Pod Autoscaler
+
+The Pod Autoscaler custom resource allows you to implement your own reconciler and autoscaling system, without changing anything else about the Knative Serving system.
+
+You can implement your own Pod Autoscaler if the requirements of your workload cannot be covered by the KPA or HPA, for example if you want to use a more specialized autoscaling algorithm or if you need to use a specialized set of metrics not support by Knative out of the box.
```

Comment:
```suggestion
You can implement your own Pod Autoscaler if the requirements of your workload cannot be covered by the KPA or HPA, for example if you want to use a more specialized autoscaling algorithm or if you need to use a specialized set of metrics not supported by Knative out of the box.
```

Reply:
Oops, my bad for copy/pasting :laughing: